rust/core-lib/src/tabs.rs

```diff
@@ -47,6 +47,9 @@ use plugins::rpc_types::{PluginUpdate, ClientPluginInfo};
 #[cfg(target_os = "fuchsia")]
 use apps_ledger_services_public::{Ledger_Proxy};
 
+/// A client can use this to pass a path to bundled plugins
```

Comment:
Should probably say "this environment variable" as it's not otherwise obvious what "this" is.

Reply:
this line is removed in #433.